### PR TITLE
Modified Node class to take as input the aci-meta.json file

### PR DESCRIPTION
**Pull request by gccarcadia**

### CODE REVIEW

File: pyaci/core.py [hunk 1]
```diff
@@ -52,17 +52,17 @@ def subLogger(name):
 aciMetaDir = os.path.expanduser(os.environ.get('ACI_META_DIR', '~/.aci-meta'))
 
 if not os.path.exists(aciMetaDir):
-    raise MetaError('Unable to find ACI meta directory {}'.format(aciMetaDir))
+    aciClassMetas = dict()
+    # raise MetaError('Unable to find ACI meta directory {}'.format(aciMetaDir))
```
Comment (kprav33n) [inline pyaci/core.py]: Can you please remove this line?

File: pyaci/core.py [hunk 2]
```diff
@@ -161,13 +161,21 @@ def _rootApi(self):
...
-    def __init__(self, url, session=None, verify=False, disableWarnings=True, timeout=None):
+    def __init__(self, url, session=None, verify=False, disableWarnings=True, timeout=None, aciMetaFilePath=None):
         super(Node, self).__init__()
         self._url = url
         if session is not None:
             self._session = session
         else:
             self._session = requests.session()
+
+        if aciMetaFilePath:
```
Comment (kprav33n) [inline pyaci/core.py]: Please change this to `if aciMetaFilePath is not None`.

File: pyaci/core.py [hunk 3]
```diff
@@ -161,13 +161,21 @@ def _rootApi(self):
...
+            with open(aciMetaFilePath, 'rb') as f:
+                logger.debug('Loading meta information from %s', aciMetaFilePath)
+                aciMetaContents = json.load(f)
+                self._aciClassMetas = aciMetaContents['classes']
+        else:
+            self._aciClassMetas = aciClassMetas
```
Comment (kprav33n) [inline pyaci/core.py]: We need to add an additional check here. If `aciMetaFilePath` is not specified, and the top level `aciClassMetas` is empty dict, we should raise an exception asking the user to specify meta properly.

File: pyaci/core.py (hunk 2 above)
Comment (gccarcadia) [reply pyaci/core.py]: Done !

File: pyaci/core.py (hunk 3 above)
Comment (gccarcadia) [reply pyaci/core.py]: Done !

File: pyaci/core.py (hunk 1 above)
Comment (gccarcadia) [reply pyaci/core.py]: Line removed.

**Pull request merged**